tests/e2e/specs/in-game/one-offs/targeting_cleanup.spec.js

```diff
@@ -53,7 +52,6 @@ describe('Clean-up of One-Off Targets', () => {
   });
 
   it('ONE-OFF Target should be removed after one-off resolves - target is FACE CARD', () => {
-    cy.skipOnGameStateApi();
```

Comment:
Same here. This test looks at the 9's one-off effect, but that has not yet been built in the GameState API. Leave this skipped for now